package.json

```diff
@@ -88,7 +88,8 @@
         "uuidjs": "^3.3.0",
         "vue": "^3.5.8",
         "vue3-gettext": "^3.0.0-beta.6",
-        "webpack-bundle-tracker": "^3.1.0"
+        "webpack-bundle-tracker": "^3.1.0",
+        "webpack-manifest-plugin": "5.0.0"
```

Comment:
this should instead be a dev dependency

Reply:
it's being used in both prod and dev builds though

Reply:
Sorry I did not realize that has any signficance with regard to dependencies vs devDependencies in Arches. With that said, do I need to make a PR into the `arches-dev-dependencies` repo?